src/dialogs/more-info/controls/more-info-light.ts

```diff
@@ -147,72 +146,66 @@ class MoreInfoLight extends LitElement {
                         class="segmentationButton"
                       ></ha-icon-button>
                     </div>
-                    
-                    ${
-                      supportsRgbw || supportsRgbww
-                        ? html`<ha-labeled-slider
+
```

Comment:
No idea why GitHub marks that many changes. I only changed/moved lines that contain `<hr></hr>` (two removals, one addition).

Reply:
You probably modify whitespace - see without whitespace changes: https://github.com/home-assistant/frontend/pull/9080/files?w=1